cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -395,15 +458,26 @@ func doInstall(mst *initramfsMountsState, model *asserts.Model, sysSnaps map[sna
 		return err
 	}
 
-	currentSeed, err := mst.LoadSeed(mst.recoverySystem)
+	if preseed {
```

Comment:
A comment about why different behavior is required for this case would be nice

Reply:
Added